jobs/awslogs/templates/bin/pre-start

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "[general]"
+echo "state_file = /var/vcap/sys/log/awslogs/agent-state"
+
+for SYSTEM_LOG in /var/log/audit/audit.log /var/log/auth.log /var/log/dpkg.log /var/log/messages /var/log/syslog
```

Comment:
Not sure I see the value in putting the static list of log files in here, why not just leave it where it is in the release today?